### ISSUE TITLE
Fail using custom router

### ISSUE THREAD

**Issue by GuillaumeBiton** [parcel-bundler/parcel#264]
🐛 bug report

how to disable href import when using custom router. I'm using Framework7 which includes it's own router ?

Regards

**Comment by DeMoorJasper** [comment]
U mean virtual routes?
This should get fixed once #224 lands
If this is about virtual routing than this is a duplicate of #211 i'll close it for now, feel free to reopen

**Comment by GuillaumeBiton** [comment]
thanks @DeMoorJasper 

Yes it's what I'm looking for but when I manually add the fix in src/assets/HTMLAsset.js it doesn't work And I can't find why. 

I have try to put it in lib also but I have node.js version 9.

**Comment by DeMoorJasper** [comment]
Could u give a lil more info about your setup?

**Comment by GuillaumeBiton** [comment]
this is how I call pages in index.html
`<ul><li><a href="/left-page-1/">Left Page 1</a></li></ul>`

in the array of my virtual router :
`{
    path: '/left-page-1/',
    url: './pages/left-page-1.html',
  },`

**Comment by DeMoorJasper** [comment]
It seems to be working fine to me?
The routing array probably doesn't get resolved with the new paths?

**Comment by GuillaumeBiton** [comment]
Even if I delete 'a' in ATTRS.href variable. It tries to bundle href references. I think it doesn't use HTMLAsset.js

**Comment by DeMoorJasper** [comment]
change parcel-bundler in ur package to "git://github.com/DeMoorJasper/parcel.git#feature/linkChecking" and run a npm install or yarn to test my fix

**Comment by DeMoorJasper** [comment]
U could also edit node_modules manually but that would take some more time

**Comment by GuillaumeBiton** [comment]
@DeMoorJasper nope doesn't work with your version too

How could I log to see which asset is used ?

**Comment by DeMoorJasper** [comment]
if
```
<ul><li><a href="/left-page-1/">Left Page 1</a></li></ul>
```
turns into
```
<ul><li><a href="/somehash.html">Left Page 1</a></li></ul>
```
Or if you are getting an error "asset can't be resolved" than it isn't my version

**Comment by GuillaumeBiton** [comment]
can't see because it fails so it doesn't give me an output. for my index.html

**Comment by GuillaumeBiton** [comment]
Users/guillaumebiton/Downloads/contacts/index.html: Cannot resolve dependency './../../../../left-page-1'

**Comment by DeMoorJasper** [comment]
Yeah than it's not using my version

**Comment by GuillaumeBiton** [comment]
don't have another install

**Comment by GuillaumeBiton** [comment]
with --no-cache err message as changed

**Comment by GuillaumeBiton** [comment]
OK this is it. uninstall your version install back one from nom and apply fix into HTMLAsset.js and with no-cache option it works

**Comment by GuillaumeBiton** [comment]
Hello, sorry for asking question here but I am using framework7.io and when I try to use it throw parcel with the internal F7 router, it fails.

First Attempt: 
Keep html page files outside bundler => fails

Second Attempts :
import html files into bundler with F7 option that allow my to name html pages.
then route using this name. => fails.

some capture:
<img width="188" alt="src_sturture" src="https://user-images.githubusercontent.com/729578/34496207-ee0fbe1c-eff8-11e7-92f5-04ebf67adf05.png">

index.html
```
<!DOCTYPE html>
<html>
  <head>
    <title>Framework7</title>
    <link rel="stylesheet" href="index.css">
  </head>
  <body>
    <div class="view view-main view-init" data-url="/"></div>
    <script src="index.js"></script>
  </body>
</html>
```

index.js
```
import Framework7 from 'framework7'

var app = new Framework7({
  routes: [
    {
      path: '/',
      // pageName: 'home',
      url: './pages/home.html',
    },
  ]
})
```

home.html
```
<div class="page" data-name='home'>
  <div class="navbar">
    <div class="navbar-inner sliding">
      <div class="title">Framework7</div>
    </div>
  </div>
  <div class="page-content">
    <div class="block">content here</div>
  </div>
</div>
```
version is 1.4.1

**Comment by GuillaumeBiton** [comment]
I update to 1.6.2 but not woking neither

**Comment by GuillaumeBiton** [comment]
OK now (1.9.0) it works with dev server but sub directory aren't bundle into list
